config/environments/development.rb

```diff
@@ -33,6 +33,8 @@
 
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
 
+  config.action_mailer.delivery_method = :letter_opener
```

Comment:
@tessereth yes, I noticed after the fact.

So letteropener runs entirely locally and when an email is sent in in the development env it opens a browser tab containing the email.

I'm open to using mailtrap but I would rather not depend on a 3rd party service unless there's other compelling reasons to do so.